contrib/static-builder-x86_64/Makefile

```diff
@@ -0,0 +1,10 @@
+RUNTIME := podman
+IMAGE := crun-builder
+
+build-crun:
```

Comment:
why not have this one depend on `build-image`?

Reply:
I'd like to cache the image so that I can reuse it.  I am thinking of pushing it to a registry and add a Travis task to verify that the static build doesn't link any shared library.